db/db/src/main/java/io/helidon/db/DbExecute.java

```diff
@@ -0,0 +1,407 @@
+ *     <li>{@code DbRowResult} {@link #query(String, Object...)} - shortcut method to unnamed query with a list of parameters
+ *     (or with no parameters at all)</li>
+ * </ol>
+ * The first three methods return a statement that can have parameters configured (and other details modified).
+ * The last two methods directly execute the statement and provide appropriate response for future processing.
+ * All the methods are non-blocking.
+ */
+public interface DbExecute {
+    /*
+     * QUERY
+     */
+
+    /**
+     * Create a database query using a named statement passed as argument.
+     *
+     * @param statementName the name of the statement
+     * @param statement the query statement
+     * @return database statement that can process query returning multiple rows
+     */
+    DbStatement<?, DbRowResult<DbRow>> createNamedQuery(String statementName, String statement);
+
+    /**
+     * Create a database query using a statement defined in the configuration file.
+     *
+     * @param statementName the name of the configuration node with statement
+     * @return database statement that can process query returning multiple rows
+     */
+    DbStatement<?, DbRowResult<DbRow>> createNamedQuery(String statementName);
+
+    /**
+     * Create a database query using a statement passed as an argument.
+     *
+     * @param statement the query statement to be executed
+     * @return database statement that can process the query returning multiple rows
+     */
+    DbStatement<?, DbRowResult<DbRow>> createQuery(String statement);
+
+    /**
+     * Create and execute a database query using a statement defined in the configuration file.
+     *
+     * @param statementName the name of the configuration node with statement
+     * @param parameters    query parameters to set
+     * @return database query execution result which can contain multiple rows
+     */
+    default DbRowResult<DbRow> namedQuery(String statementName, Object... parameters) {
+        return createNamedQuery(statementName).params(parameters).execute();
+    }
+
+    /**
+     * Create and execute a database query using a statement passed as an argument.
+     *
+     * @param statement  the query statement to be executed
+     * @param parameters query parameters to set
+     * @return database query execution result which can contain multiple rows
+     */
+    default DbRowResult<DbRow> query(String statement, Object... parameters) {
+        return createQuery(statement).params(parameters).execute();
+    }
+
+    /*
+     * GET
+     */
+
+    /**
+     * Create a database query returning a single row using a named statement passed as an argument.
+     *
+     * @param statementName the name of the statement
+     * @param statement the statement text
+     * @return database statement that can process query returning a single row
+     */
+    DbStatement<?, CompletionStage<Optional<DbRow>>> createNamedGet(String statementName, String statement);
```

Comment:
Another occurence of _hybrid_ method with both statement name and statement string.